.maintain/optimize-cache.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+#
+
+set -x
```

Comment:
`set -xe` might be a better choice (stops running the following commands if any error occurs).

Reply:
No, If it is the first time to execute, it is possible that the directory does not exist or there is no compiled package, so it is allowed to fail here

Reply:
Then you might want to add some bash parameter that makes the shell always return the exit code 0, or explicitly ignore the exit code from the caller side like `./optimize-cache.sh || exit 0`.